### DISCUSSION TITLE
Welcome to OnBoarding Discussions!

### DISCUSSION THREAD

**Discussion by JuanRodiles** [Launch-X-Latam/OnBoarding#1] [Announcements]
## 👋 ¡Bienvenidxs!
  Estamos utilizando Discusiones como un lugar para conectar con otros miembros de nuestra comunidad. Aquí podrás:
  * Plantear tus dudas.
  * Compartir ideas.
  * Relacionarte con otros miembros de la comunidad.

Para empezar, comenta abajo con una presentación tuya y cuéntanos qué haces en esta comunidad.

**Comment by jorgesniper** [comment]
hola! me llamo jorge, tengo 22 años, y estoy aquí para aprender a desarrollar

**Reply by BrayanGil090720** [reply]
> hola! me llamo jorge, tengo 22 años, y estoy aquí para aprender a desarrollar

que onda carnal ✌️

**Comment by gerardolggr** [comment]
Hola, me llamo Luis, tengo 22 años, soy de Guanajuato y estoy aquí para ampliar mi conocimiento en programación.

**Comment by Vitaliquid** [comment]
Hola a tod@s!

Soy Ricardo (Vital Liquid), estoy muy emocionado de pertenecer a esta organización y quiero aprovechar mi tiempo de la mejor manera para aprender lo más posible. Espero de esta comunidad poder hacer las conexiones que me permitan aportar ayudar a crecer y crecer mis conocimientos sobre tecnología y desarrollo web. ánimo!

**Comment by Branmedlo** [comment]
Saludos!! Soy Brandon, vivo en Jalisco. Aprovecharé la oportunidad para aprender sobre el desarrollo web, aunque es algo que me gusta y he aprendido algunas cosas como autodidacta ahora se abre una buena oportunidad.

**Comment by JuanRodiles** [comment]
# ¡Hola a todas y todos!

Es un gusto tenerlos por acá. Cuéntenos más sobre ustedes.

No olviden realizar el **OnBoarding**. Al finalizar esta semana les daremos el acceso a Microsoft Teams donde accederán a las sesiones en directo. 

Recuerda:

- [ ] Darte de alta en GitHub.
- [ ] Revisar los tutoriales.
- [ ] Sumarte a la Organización.
- [ ] Completar tu acceso a Microsoft Teams al final de la semana.

**Reply by JuanTHz** [reply]
Hola realizando los pasos del Onboarding llegando al paso _"Etapa 2: Unirte a la organización de Launch X LATAM"_  al ingresar al portal Entrar a [https://launchxauth.azurewebsites.net/](https://launchxauth.azurewebsites.net/joingh) manda el mensaje de que el sitio está detenido y no puedo completar ese paso 
![image](https://user-images.githubusercontent.com/114203252/191884148-72ac4887-6af9-44d4-b324-beeecac047a2.png)

**Reply by ittzel** [reply]
Hola!!! Pudiste resolverlo? Es que me aparece lo mismo y no he podido solucionarlo

**Reply by JuanTHz** [reply]
> Hola!!! Pudiste resolverlo? Es que me aparece lo mismo y no he podido solucionarlo

No aun no, en lo que responden puedes continuar con los siguientes pasos.

**Reply by JuanTHz** [reply]
@ittzel ya está disponible 
![image](https://user-images.githubusercontent.com/114203252/191902382-3c610967-3553-4d40-96eb-de1cfb2f73ba.png)

**Comment by BrayanGil090720** [comment]
hola, me llamo Brayan De Lira, **soy hermano gemelo**  😮  de @jorgesniper , por lo tanto tambien tengo 22 años casi 23 😃  XD, es un gusto para mi estar en esta travesia 💯 , y les deseo mucha suerte a todos 👍 .

**Comment by Jonathan-12-LA** [comment]
Hola , un cordial saludo a todos me llamo Jonathan Larin  soy de El Salvador  mi intencion es  aprender todo lo que sea posible para ampliar y mejorar mis conocimientos y habilidades en la programacion y el desarrollo web.

**Reply by supernova945** [reply]
Que onda

**Comment by nabimeyz** [comment]
Buen día a todos, parece ser que soy la primer mujer en comentar en la discusión.

Me llamo Metztli, soy de CDMX, tengo 20 años y estoy cursando una Lic. En Informática. Quiero ampliar mis conocimientos en frontend y backend con Java. Esperemos que esta oportunidad sea muy enriquecedora para todos nosotros. 👩🏻‍💻💫

**Reply by MauZubi** [reply]
Bienvenida nabimeyz, somos 2 para ampliar nuestros conocimientos y ponerlos en practica.

**Reply by Amhedriel** [reply]
Buenas **nabimeyz** me agrada tu entusiasmo, pero te comento que veremos ***JavaScript*** hasta donde tengo entendido y no ***Java***, te informo que son lenguajes muy diferentes, es más, no tienen nada que ver el uno con el otro, lo menciono para evitar confusiones, saludos.

**Comment by ArturoRodriguezF5** [comment]
Saludos a todos, me llamo Arturo Rodríguez Flores, soy de México y tengo 28 años, soy egresado de ingeniería en computación, y estoy muy emocionado por reforzar conocimientos en desarrollo, y en iniciarme como full stack developer. 😼

**Comment by LuceroLuciano** [comment]
Hola a todxs! 
Soy Lucero, me gusta mucho la comunidad porque es una forma increíble de aprender de manera divertida y en compañía de  muchxs explorers es mejor! 

¿Alguien me podría orientar sobre como hacer el onboarding?, seguí todas las instrucciones y el botón del onboarding que dice: **Accede dado click aqui**, me trajo a este repositorio pero no estoy segura sobre si es todo lo que se tiene que hacer o me falta realizar algo. 
Agradecería mucho su ayuda.  

Saludos explorers.

**Reply by Branmedlo** [reply]
Hola @LuceroLuciano al parecer ya todo está hecho, me fijé en tu perfil que ya perteneces a la organización que ese era uno de los pasos más importantes. Por lo demás sería ver los tutoriales y familizarnos con Github y Team.

**Reply by JosafatJimenezB** [reply]
Hola, que gusto encontrar a un ajolonauta!!! woopa!!

**Comment by sacreddog** [comment]
Hola a todos 

Estoy muy contento de ser parte del equipo.

Síganme para conversar y aprender más.

**Comment by isbelangy** [comment]
**Hola**✌️🙂

Soy Mara, diseñador gráfico de la CDMX. 

Aunque mi acercamiento con el desarrollo no es muy basto, estoy muy entusiasmada con el curso.

(^_-)

**Comment by DisExcpt** [comment]
Hola que tal, mi nombre es Axel tengo 22 años , estudio una ing en computación y estoy emocionado por estar aquí.
El motivo por el cual me encuentro aquí es para ampliar más mis conocimientos sobre programación web.

**Comment by D3r3-k** [comment]
¿Qué tal? Mi nombre es Derek, tengo 22 años, soy de Guatemala, actualmente estoy estudiando una ingenieria en ciencias y sistemas y estoy emocionado por aprender y ampliar mi conocimiento sobre full stack! 
Espero llevarme bien con todos!

**Comment by EinarAliagaG** [comment]
Hola; espero esten muy bien, me llamo Einar tengo 23 años y estoy muy emocionado de empezar este viaje de aprendizaje. Les deseo exito a todos y que lleguemos al final de nuestro viaje.

**Comment by federico42o** [comment]
Hola!!
Como andan? Mi nombre es Federico soy de Argentina, estoy muy contento y es un gusto estar acá! Muchos éxitos a todos y ojala aprendamos muchas cosas juntos 😁

**Comment by Cueto93** [comment]
Hola a todos,
Mi nombre es Julio Cueto tengo 29 años, acabo de terminar mi maestria en Gestion de Tecnologías de la Información y estoy aqui porque quiero ampliar mis conocimientos de FronEnd y BackEnd.  👍 
Saludos

**Comment by Amhedriel** [comment]
# ¡Hola gente linda!

Muy buenas a todos, soy David Amhed Beltrán R. de C. y estoy acá para aprender todo lo posible, debido a mi trabajo y al cuidado de mi familia el tiempo que puedo dedicar es algo corto, sin embargo trataré de aprovecharlo al máximo en esta comunidad. 

Al ver el curso trazado me interesa mucho el *desarrollo web full stack* y me intriga el uso de **JavaScript** para el desarrollo en *BackEnd*, yo tenía entendido que se utilizaba ***Python*** más comúnmente en este tipo de desarrollo, pero, al ver algo relacionado de *"Microsoft"* creí que usaríamos ***C#*** o ***.NET***, sin embargo estoy muy emocionado por ver todo lo que aprenderemos y no puedo esperar hasta mañana, que dará inicio este nuevo camino de aprendizaje, muchas gracias por dejarme se parte de esto.

Saludos gente linda.

---

**Comment by iVanOoOvich** [comment]
Hola a todos!
Soy de México, tengo 34 años y estudié ingeniería en comunicaciones. Estoy muy emocionado por ampliar mi conocimiento en la programación.

**Comment by karenzum** [comment]
Hola !!
Un saludo a todos! Soy Karen y soy de México.. Les deseo éxito a todos!

**Comment by ittzel** [comment]
**Hola a todos!!! **

Mi nombre es _Itzel_ soy de Veracruz, México. Y me inscribí a este curso para expandir mis conocimientos sobre desarrollo frontend y backend. Estoy muy emocionada por comenzar deseo que sea una etapa muy enriquecedora, llena de conocimientos y de éxito para todos✨👩🏻‍💻

**Comment by AGrillin** [comment]
¡Hola! ¿Cómo están?
Soy Aarón de Chihuahua, me inscribí para aprender sobre desarrollo frontend y backend ya que quiero ampliar mis conocimientos. 
Éxito a todos.

**Comment by ElverAndrew** [comment]
¡Hola a todos!
Mi nombre es Andrew y soy de Perú, tengo 18 años y estoy estudiando Ingeniería de Software y estoy muy emocionado por esto y por adquirir nuevos conocimientos.

**Comment by D3r3-k** [comment]
Tengo una duda, en el mapa de viaje del programa, dice que participemos en el hackday, el hackday es la inauguración o es otra cosa?
Y otra duda que tenia era como funcionara lo del examen de microsoft azure?

**Reply by federico42o** [reply]
Tengo entendido que el Hackday es en los últimos días del programa

**Reply by D3r3-k** [reply]
oh ya, muchas gracias! @federico42o

**Reply by zoearenasc97** [reply]
Lo del challenge de Azure cómo les comentamos es opcional y también se los iremos dando mientras avanza el programa. 🥇

**Comment by WhoPotz** [comment]
Hola me llamo Edgar y tengo 22 años y espero con ansías aprender muchas cosas. 💯💯💯

**Comment by falcongmz** [comment]
Hola a todos me llamo Carlos y quiero seguir aprendiendo de este mundo.!! exitos a todos.

**Comment by gumalo007** [comment]
Buen día soy de Tabasco Mex. Y quiero aprovechar esta gran oportunidad

**Comment by Juan4916** [comment]
Hola buen día me Llamo Juan Carlos  Dávila estoy aquí para aprender y adquirir mucho conocimiento .

**Comment by zusldev** [comment]
Hola soy Liborio y es mi primera vez en un programa como este

**Comment by imap07** [comment]
Hola team, soy Carlos Mendoza, me inscribi a este curso para fortalecer conocimientos en back y front. Saludos!!!!

**Comment by XplorerJM** [comment]
Hola a tod@s! Soy Jair Mtz. de CDMX, estoy aquí para ampliar y reforzar mis conocimientos. 
Un gusto saludarles!  🤟

**Comment by Adamthefirstmantogetlaid** [comment]
Bitcoin or Eth

**Comment by Ivanhdzsty** [comment]
Hola buen dia, soy Ivan busco adquiri mas conocimiento, aprender cosas nuevas y poder aplicarlas

**Comment by edesk22** [comment]
Saludos a todos, me llamo Edgar y soy de la Ciudad de México. Me inscribí en este proyecto para aumentar mis conocimientos y reforzar algunas cosas que ya sé, espero que todos nos podemos divertir y aprender.

**Comment by zoearenasc97** [comment]
Qué ondaaaaaaaaa! 🙊
Estoy muy contenta de ver gente de todo LATAM :')
¿Sabían que soñabamos con este momento? <3 

**Ya quiero conocerlos a todos** 💖💖💖💖💖

Les deseo mucho éxito hoy y siempre. 
Cualquier duda ando en el instagram de Innovaccion Virtual o en el correo junto a Naye, qué también me ayuda mucho a contestarles.🙊

![image](https://user-images.githubusercontent.com/87114334/192062249-a37706c2-8bba-4d1c-9bbb-e6a85135d928.png)

**Comment by Auv0Dev0** [comment]
Hola, mi nombre es Auvis. Soy de Venezuela y tengo 20 años. Estoy estudiando Ingenieria Informática, estoy aquí porque quiero seguir aprendiendo y para profundizar conocimientos.

**Comment by MacaCoronel93** [comment]
Hola a todos ! increible conocerlos

**Comment by AnViCa95** [comment]
Hola buenas tardes. Me llamo Angel soy de Veracruz, México. Estoy aquí porque quiero aprender nuevos conocimientos y poco a poco deslindarme del área laboral donde he pasado mis últimos 9 años laborando. Tengo 27 años.

**Comment by ManueLuna-Mx** [comment]
Hola a todos, mi nombre es Manuel, espero que todos tengamos mucho éxito en esta nueva campaña.

**Comment by ghost** [comment]
Hola, me llamo Felipe. Tengo 21 años. Soy de México. Soy estudiante de Ingeniería en Tecnologías de la Información. Me inscribí en este curso, porque quiero adquirir nuevas habilidades en el desarrollo Frontend y Backend.

**Comment by msaldivar-dev** [comment]
Buenas a todos

Me llamo Marcelo, tengo 23 años, soy de Paraguay, estoy estudiando ingeniería en informática, me encuentro aquí para ampliar mis conocimientos
Estoy más interesado en el backend pero también con ganas de saber de frontend

**Comment by junae67** [comment]
hola un gusto me llamo juan pablo Vargas 22 años vivo en Colombia Bogotá

**Reply by ronalc90** [reply]
Un gusto, he visto algunos de tus trabajos :)

**Comment by MartiRF** [comment]
Hola soy martin, vivo en mexico y estes feliz de participar c:

**Comment by Promi1199** [comment]
Hola gente, me llamo Miguel Angel, estoy muy entusiasmado de comenzar este viaje con ustedes.

**Comment by JoshuaPerez22** [comment]
Hola a todos. Yo soy Joshua Pérez, soy estudiante de la carrera Matemáticas Aplicadas y Computación. Estoy emocionado de participar en esta edición de Launch X, ya que ya tuve una buena experiencia anteriormente con Inovacción Virtual. Mucho éxito a todos.

**Comment by AbiKorzyk** [comment]
Hola, soy Abigail. Soy estudiante de la carrera Ingeniería en Tecnologías de la información. Me uní a la comunidad de Launch X para ampliar mis conocimientos, estoy muy interesada en aprender.

**Comment by FedericoF03** [comment]
Buenas, me llamo Federico Fernández tengo 21 años y estoy estudiando de forma autodidacta desde 7 meses aprox. Emocionado por adquirir experiencia mas comunicatoria con todos!

**Reply by David-AC** [reply]
Que increible! Bienvenido!

**Comment by AnaPatino18** [comment]
Hola a todxs, buenas tardes, mi nombre es Ana, soy de Acámbaro Guanajuato y estoy en esta misión para aprender muchooooooooooooo ;)

**Comment by enKrypt0** [comment]
Hola a todos, soy Diego Vélez, tengo 17 años, soy de Ecuador y estoy aquí para desarrollar mis habilidades con ayuda de este gran proyecto, gracias a mi interés por la programación me dio la idea de ingresar a este curso y aprender mucho mas.
Mis intereses son desarrollarme en todos los ámbitos, tanto en back-end como en front-end, también me interesa la ciberseguridad y el hacking ético, pero por ahora me gustaría aprender en este área.

**Reply by David-AC** [reply]
Bienvenido!

**Comment by Del4xuw** [comment]
Me llamo Kevin Torrez soy estudiante de ingeniería en sistemas. Me interesa aprender

**Comment by AngelaVanessa** [comment]
Hola, mucho gusto con todos mi nombre es Angela, tengo 26 años y estoy emocionada por comenzar esta nueva experiencia :)

**Comment by SageApolo40637** [comment]
Hola me llamo Eduardo, tengo 22 años y soy de CDMX, entre con la finalidad de aprender más y conocer a gente para compartir opiniones. Mucha suerte a tod@s

**Comment by juanmadaz** [comment]
Hola! Soy Juan Dávila, tengo 31 años y soy Ingeniero en Mecatrónica. Estoy en éste curso para aprender y complementar lo que hago en mi trabajo para poder seguir creciendo laboralmente. Saludos!

**Comment by Yerrigg** [comment]
Hola, Soy Yerri tengo 20 años. Soy estudiante de Ingenieria de Sistemas. Estoy muy contento de unirme a las misiones de Launch X espero seguir aprendiendo. Un gusto con todos!.

**Comment by Matias-Py** [comment]
Hola a todos me llamo Matias tengo 22 años soy de Buenos Aires y estoy muy feliz de poder estar acá y poder aprovechar esta oportunidad

**Comment by viridianab** [comment]
Hola, me llamo Viridiana, soy diseñadora gráfica :^), de B.C, México. Participé en la edición anterior y aquí nuevamente para seguir aprendiendo. 😄

**Reply by JosafatJimenezB** [reply]
hola explorer, recuerdo que habia visto tu pokedex en una discussion

**Comment by ChablePJA** [comment]
Hola a todos. Mi nombre es José Chablé y tengo 21, soy de Oaxaca, México. Decidí entrar a LauchX con la finalidad aprender junto a una gran comunidad y ampliar mis habilidades adquiridas como estudiante de la Ingeniería en comunicación multimedia

**Comment by SaulMMBP** [comment]
Hola, me llamo Saul, tengo 27 años, espero aprender mucho de este viaje y de la comunidad.

**Comment by MCurtis7** [comment]
Hola a todos, disculpen saben que día comienza oficialmente el reto con la primera misión y clase? Buen día.

**Reply by XplorerJM** [reply]
Tengo entendido que hoy Lunes 26, échale un vistazo al grupo de Teams.

**Comment by chaconrocio** [comment]
Hola a todos!! Mi nombre es Rocío Chacón tengo 20 años y soy salvadoreña, estudiante de primer año de ingeniería en sistemas, estoy muy feliz de pertenecer a esta gran comunidad, espero aprender mucho junto con todos ustedes.

**Comment by fabiancgc12** [comment]
Hola a todos, soy Fabian, soy Ing. en informática y estoy aquí para ampliar mis conocimientos sobre el desarrollo web.

**Comment by MiguelHSV** [comment]
Hola buenas mi nombre es Miguel y nada darles las gracias por la oportunidad que nos estan dando a todos para seguir desarrollando nuevas habilidades.

**Comment by David-AC** [comment]
Hola a todos! Mi nombre es David Aguilar, soy de la CDMX, llegué aquí porque ya había participado previamente en el ciclo de Innovacción AZ-900. Actualmente estoy terminando mi licenciatura en Biotecnología y mi objetivo es poder conectar mi carrera con lo que aprenda aquí.
¡Es un gusto poder estar aquí con todos ustedes!

**Comment by ghost** [comment]
Hola mi nombre es Edgar espero aprender muchas cosas y pasármela bien!!!

**Comment by ErickOrtiz0298** [comment]
Mi nombre es Erick Ortiz, tengo 24 años, soy ingeniero en Mecatrónica, tengo muchas ganas de desarrollarme como full stack y creo que esta será una increíble experiencia.

**Comment by YerValMoCa** [comment]
mi nombre es Yerry Valentin Morales Carranza, un poco largo verdad! Podes llamarme Yerval Moca, una unión entre mi nombre., soy de Costa Rica y Pura Vida a todos, los veo el lunes 26 a las 7pm, buenas noches a todos y disfruten fin de semana. Chao

**Comment by alchemyyt** [comment]
Hola me llamo jose montes de oca soy de venezuela y estoy aqui para mejorar mis conocimientos de desarrollo web mucho gusto. 👋

**Comment by Dsandres22** [comment]
¡Hola mundo! Mi nombre es Andres Dussan, soy de Bogotá-Colombia y estoy acá con muchas ganas y expectativas de aprender de este maravilloso mundo

**Comment by K3NNY6** [comment]
Hola mi nombre es Carlos Alberto me pueden llamar Carlos o Alberto. Estoy muy emocionado de comenzar esta aventura espacial junto con todos ustedes espero lograr comprender todo le voy a dedicar mucho por que siendo sincero vengo aqui sin ningun tipo de conocimiento de programación, Actualmente laboro en un Taller de Reparación de equipós de computo, impresoras y laptops asi que en lo que les pueda apoyar con muchisimo gusto. La razón por la cual me registre es por que quiero adquirir todo el conocimiento posible sobre desarrollo web.

**Comment by Hernanjas400** [comment]
Hola buenas noches!!!!! mi nombre es Hernán soy de Bahía Blanca, Argentina con muchas ganas de aprender y compartir con este maravilloso grupo Saludos para todos.

**Comment by EdgarTEspinosa** [comment]
Hola mi nombre es Edgar Trejo Espinosa soy del Estado de México, tengo 43 años. Gracias por aceptarme.

**Comment by MineAlvarezDCA** [comment]
Hola! 
Mi nombre es Minerva, me parece una oportunidad maravillosa el poder entrar a esta aventura de programación 😄 👨‍🚀 🚀 
Pero... alguien me podria orientar?, no he podido unirme a la organización, coloco mi correo(con el que hice esta cuenta de github), y me dice "correo enviado" pero en la bandeja de entrada no aparece nada. Estare haciendo algo mal? 😟 ❓

**Reply by iVanOoOvich** [reply]
Hola Minerva, un gusto! Estás colocando el correo que te proporcionaron @innovaccion.mx? Si es así revisa en tu bandeja de no deseados posiblemente ahí esté. Espero que lo logres, saludos.

**Reply by MineAlvarezDCA** [reply]
Hola! 
Si, la cuenta de github la hice con el correo de innovaccion asi que ese es el que estoy colocando.
De momento no me aparece nada en la bandeja normal o la de no deseados, esperare un poco haber si llega. Muchas gracias!!

**Reply by ManuOSMx** [reply]
¡Hola @MineAlvarezDCA !
Hemos alcanzado el límite diario por parte de GitHub en enviar invitaciones para unirse a la organización de LaunchX LATAM en GitHub. Por esta razón no te ha llegado la invitación.
Mantente al pendiente de nuestras redes sociales donde anunciaremos que ya se habilitó la página y puedas intentarlo nuevamente.
Solo verifica escribir bien el correo al que se va a mandar la invitación. 

Por el momento puedes continuar con llenar el formulario usando el correo de Innovacción y esperar a ser aceptado en el equipo en Teams.

Cualquier duda, estaré al pendiente.

Un saludo. 😊

**Reply by MineAlvarezDCA** [reply]
Entendido, estaré al pendiente! 😄 👍

**Reply by ManuOSMx** [reply]
¡Hola @MineAlvarezDCA !

Ya se encuentra habilitada la página para que puedas intentar nuevamente unirte a la organización de Launch X LATAM en GitHub.
El enlace es: [launchxauth.azurewebsites.net/joingh](https://launchxauth.azurewebsites.net/joingh)

Por favor, revisa que el email que coloques esté bien escrito antes de darle en "enviar invitación".

Al enviar la invitación, el correo para que te unas a la organización no debe de tardar más de 5 min. en llegarte al email que ingresaste en la página.

Cualquier duda, estaré al pendiente.
Ten un excelente fin de semana.

**Comment by EdgarTEspinosa** [comment]
¿ No es el correo con el que creaste la cuenta de github.com? A mi tampoco me llega todavía

Obtener Outlook para Android<https://aka.ms/AAb9ysg>
________________________________

**Reply by iVanOoOvich** [reply]
Hola! Tienes razón es el correo con el que creas tu cuenta GitHub, me confundí, espero les llegue el correo! Saludos.

**Reply by Murillas** [reply]
Mi GitHub es el personal, debí haber hecho uno con el correo que nos enviaron?

**Reply by pjdiaz9731** [reply]
Correcto el GitHub lo hice con mi cuenta personal, pero para poder acceder a las misiones y el Teams me solicita unos datos de acceso que no coinciden con los que yo hice el registro.

Me imagino que deben de ser los datos de acceso que brindan ellos, pero sigo sin recibirlos.

**Reply by federico42o** [reply]
No te llego un mail con el calendario y las instrucciones? ahi estaban los datos de acceso

**Reply by pjdiaz9731** [reply]
No amigo todavía estoy al pendiente de esos datos de acceso.

**Comment by FrancoFlowers** [comment]
Hola! Mi Nombre es Franco Flores, soy de Nuevo León, México es un gusto poder estar aquí con todos ustedes.

**Comment by Emmanueljn88** [comment]
Hola a todes, soy Emmanuel y como alguna vez dijo Videgaray: vengo a aprender :P

Saludos

**Comment by DiegoHC06** [comment]
¡hola! comunidad, me llamo Diego,  soy de Yucatán, me uní a la comunidad virtual para reforzar mis conocimientos y aprender cosas nuevas. saludos

**Reply by XplorerJM** [reply]
Bienvenido y saludos desde CDMX!

**Comment by angeltorrez** [comment]
Hola buenas!
me llamo Angel y tengo 23 años me encantaría poder no solo aprender como piensa un Full Stack, sino el trabajar en conjunto como comunidad para llegar a ese mismo destino, me siento entusiasmado, gracias y saludos.

**Comment by MateoB4** [comment]
Hola, soy Mateo de Cali-Colombia,de profesión ingeniero electrónico y con muchas expectativas alrededor de LaunchX y todo lo que nos puedan compartir con esta gran oportunidad.

**Comment by pjdiaz9731** [comment]
Hola qué tal, tengo una duda respecto al la cuenta que se nos asigna en el correo de bienvenida, ya que este no me ha llegado y no he podido acceder al teams.

Alguien podría ayudarme o decirme si algo está mal .
Saludos,

**Comment by oramirezj** [comment]
Buen día mi nombre es **Omar Ramírez**, tengo 42 años, actualmente soy optometrista, debido a la pandemia estoy incursionando en el mundo de la informatica, llevo estudiando Ingenieria en Sistemas desde febrero del 2020 y he tratado de aprender lo más posible intentando cambiar mi situación laboral. Un gusto compartir con personas de distintos lugares, yo soy de México vivo en el estado de Michoacán, estare tratando de aprender lo más posible y si en algo puedo apoyar también lo hare con gusto.

**Comment by MagsPalh** [comment]
Soy Laura, tengo 27. Soy de Ciudad de México, actualmente por mi carrera y durante la pandemia me incursioné a las IT.
Así que espero aprender lo más posible de esta comunidad y poder acoplar estos conocimientos con mi licenciatura.

**Comment by ckacy01** [comment]
Hola a todxs, soy Jorge tengo 18 años. Soy de Sinaloa y actualmente estoy cursando la carrera de TI.

**Comment by Ezcainer** [comment]
Hola a todos los presentes !
Me presento me llamo Pablo Zumba mi meta profesional es convertirme en un programador , gracias al apoyo de esta comunidad al recibirme estoy con todas las ganas de aprender para volverme un full stack 💪🏻 💻

**Comment by saulsavh** [comment]
Hola a todos!
Soy Saul Villalvazo, actualmente estoy por concluir la carrera de Ing. Mecatronica, Muchas gracias por la oportunidad para iniciarnos en el mundo del full stack.

**Comment by Emmanueljn88** [comment]
Hola a todos, les deseo lo mejor para estos meses de mucho aprendizaje. Me llamo Emmanuel y espero podamos acompañarnos y superar este reto.

**Comment by Javicho9804** [comment]
Hola a tod@s , mi nombre es Javier , tengo 24 años y soy de Coahuila , estoy aqui para aprender sobre desarrollo web, ya que siempre me llamo la atencion mas nunca le dedique el tiempo. 
Les deseo mucho exito a todos!

**Comment by 4rthas** [comment]
Hola soy alain de mexico y estoy para aprender más como herramienta profesional

**Comment by Diegonzaleznavas** [comment]
Saludos  y éxitos a todos soy Diego de Maracaibo, venezuela, estoy aquí para aprender más sobre desarrollo web.

**Comment by CarloAyala** [comment]
Hola a todos, me llamo Carlo, vivo en México y actualmente estoy estudiando la carrera de Mecatrónica, estoy aquí para ampliar mi conocimiento en programación y aplicarlo en mi área de estudio. **Saludos a todos y mucho éxito!**

**Comment by MiltonRodriguez37** [comment]
Hola a todos, mi nombre es Milton y soy de Guadalajara, México. Estoy aquí para desarrollar mis habilidades en desarrollo web.

**Comment by gramoslt** [comment]
Hola a todos, me llamo Gerardo, soy estudiante de ing. en computación y me interesa aprender de desarrollo web

**Comment by gerordu** [comment]
Hola a todos, mi nombre es Gerardo y me da mucha emoción poder participar en este tipo de programas. 
Comencé a programar hace unos años, sin embrago, fue un hobby que poco a poco fui dejando debido a ocupaciones, es por ello que vuelvo al camino y que mejor de la mano de Innovaccion. 🤝🏽

**Comment by rugr0n** [comment]
Hola mucho gusto soy Juan José, soy de Colombia. Me gustaría poder aprender más sobre programación y mejorar mi nivel cómo programador.

**Comment by Kerurbaez** [comment]
Hola, me llamo Kerbin y soy de Venezuala, feliz de ser parte de esta comunidad y de poder aprender juntos a ustedes, estoy animado a poder formar nuevos conocimientos y ponerlos en prácticas

**Comment by NeoCortex21** [comment]
Hola a todos mi nombre es Neo Anglada, saludos a todos. 🥸

**Comment by JosLuisDev** [comment]
Mi nombre es José Luis tengo 24 años soy de México tengo un poco de conocimiento en Java y Spring y me encantaría ser desarrollador full stack y dedicarme a esta profesión que tanto me gusta! 🥺

**Comment by iguelAL** [comment]
Hola

**Reply by XplorerJM** [reply]
Hola!

**Comment by iguelAL** [comment]
Mi nombre es miguel, soy de CDMX, tengo 21 años y no se cómo utilizar bien github, soy nuevo.

**Reply by OscarHub90** [reply]
Hola mi nombre es Oscar, soy de Bogotá Colombia y estoy muy emocionado por ser parte de esta gran experiencia, espero sacarle el mayor provecho no solo a nivel de aprendizaje sino también a nivel de conocer personas igual de entusiastas y creativas.

Saludos,,

**Reply by XplorerJM** [reply]
Hola xplorer, bienvenido. Te recomiendo revisar el contenido del OnBoarding, aquí una liga con lo básico de GitHub
Saludos!

**Comment by Dargo7** [comment]
Hola a todos, Soy Dario Gomez me conecto desde Colombia y es un gusto participar de esta experiencia de aprendizaje.

**Comment by devonent** [comment]
Hola, soy Darien Pérez Cano, estudiante de TI en la Universidad Politécnica de Tlaxcala, y sera un gusto participar en este Launch X para poder ampliar mis conocimientos en desarrollo web, así como para conocer nuevas personas y desarrollar buenos proyectos en equipo. 😄

**Comment by DannyPalma22** [comment]
Hola buenas noches , aún no me puedo unir al grupo de Microsoft Teams,alguien me podría ayudar por favor .

**Reply by devonent** [reply]
Si ya te llegó el correo por parte del grupo de Innovacción, probablemente este tutorial pueda servirte: https://www.youtube.com/watch?v=pmoQ0gzqYM4

En caso de que aún no cuentes con tu correo (en el cual vienen tus credenciales de acceso), o si tuviste algún problema con el inicio de sesión, puedes escribir un correo a contacto@innovaccion.mx y/o soportelaunchx@innovaccion.mx para que puedan atenderte. Ten en cuenta que estan atendiendo a demasiados usuarios, así que si no te responden en unos días, no te preocupes, seguramente están haciendo todo lo que pueden, se paciente y permanece atenta a los nuevos correos que te hagan llegar. Ojalá y se resuelva pronto tu inconveniente. 😃

**Reply by Amhedriel** [reply]
Bueno Danny agrego que yo tuve un problema similar con mi cuenta y envié un correo a [contacto@innovaccion.mx](mailto:contacto@innovaccion.mx "correo de soporte") y lo resolvieron en algo más de 2 días, si llenaste las formas con tus datos correctamente debería llegarte un correo en estos días con tus credenciales de acceso a Microsoft Teams, en caso de no haber llenado correctamente las formas, comunícate con soporte y pide corroborar tus datos en caso de que se aplique, suerte.

**Comment by ShaneMS2098** [comment]
Hola, Soy Shantal Silva, estoy aquí para aprender algo nuevo que desde siempre me ha llamado la atención

**Comment by Liz0798** [comment]
Hola soy Lizeth Hernandez tengo 24 y me encanta aprender cosas nuevas.

**Comment by ghost** [comment]
**Saludos Camarad@s**

Mi nombre es Aldo, vivo en CDMX y actualmente estoy laborando en el area de diseño y sistemas en una empresa que se dedica a las artes gráficas.

Es un gusto y placer poder participar y estar aquí para compartir conocimiento. Les deseo éxito a todos y seguimos en contacto.

**Comment by SilverWolfG** [comment]
### Saludos a Todos 👋 

Mi nombre es Gerson, soy de la CDMX y soy Ing. en Biotecnología, sin embargo he comenzado mi acercamiento a la programación debido a mi área laboral actual y me gustó, por lo que después de tomar algunos cursos y aprender un poco de VBA por mi cuenta comencé a seguir estos temas y llegue hasta aquí.

Literalmente llegue desde cero pero espero poder ponerme al día con todo el conocimiento que vamos a compartir en este espacio.

Es un gusto para mi poder aprender de todos ustedes y pues bueno quedamos en contacto.

**Comment by DiegoHC06** [comment]
## Hola Explorers! 

un gusto saludarlos, me conecto desde el estado de Yucatán, México, me encanta aprender cosas nuevas.

**Les deseo mucho éxito a todos**

**Comment by Ortalfc** [comment]
### Disculpen la tardanza
Hola soy Ricardo Orta de Querétaro, Mexico, voy llegando un poco tarde, pero con todas las ganas, saludos.

**Comment by MichelDHdz** [comment]
Perdón el retardo

Mi nombre es Michel vengo llegando tarde pero planeo ponerme al corriente lo más rápido posible, soy de ciudad de México, y tengo altas expectativas del curso.

**Comment by Rosruiz** [comment]
Hola, me llamo Rosa María, soy del estado de BCS y estudio gestión industrial, estoy muy emociona por aprender en este curso.
Saludos

**Reply by Adamthefirstmantogetlaid** [reply]
Hi

**Comment by ramiros34** [comment]
Saludos busco un programador para que me crea una extensión en Amazon
